x-pack/plugins/cases/public/containers/api.ts

```diff
@@ -189,7 +189,7 @@ export const getCases = async ({
 }: FetchCasesProps): Promise<AllCases> => {
   const query = {
     reporters: filterOptions.reporters.map((r) => r.username ?? '').filter((r) => r !== ''),
-    tags: filterOptions.tags.map((t) => `"${t.replace(/"/g, '\\"')}"`),
```

Comment:
Why we removed that?

Reply:
i think somehting must have changed in the EuiTable component because it doesnt required this filtering anymore, and using it will break filtering entirely. i tested a bunch of weird tags and they all work:
<img width="363" alt="Screen Shot 2021-06-09 at 10 40 42 AM" src="https://user-images.githubusercontent.com/6935300/121394983-28bb7b80-c90f-11eb-9413-9b6be1cece35.png">